src/handlers/generate/ssh/generate-ssh-config.command-builder.ts

```
@@ -16,5 +17,5 @@ export function generateSshConfigCmdBuilder(yargs: yargs.Argv<{}>) : yargs.Argv<
             description: 'Specifies an alternate location for the BastionZero config file'
         })
         .example('$0 generate sshConfig', 'Create and link an ssh config file based on your organization\'s policies')
-        .example('$0 generate sshConfig --mySshPath path/to/config --bzSshPath path/to/bz-config', `Optionally specify filepaths (defaults to ${process.env.HOME}/.ssh/config and ${process.env.HOME}/.ssh/bz-config respectively)`);
+        .example(`$0 generate sshConfig --mySshPath ${path.normalize('path/to/config')} --bzSshPath ${path.normalize('path/to/bz-config')}`, `Optionally specify filepaths`);
```

I removed the defaults from this line because I though it was misleading, we should have the defaults set in the objects above instead of keeping them as nil but that breaks system tests for some reason. I will figure out how to fix that in a different PR.